src/dmd/frontend.d

```diff
@@ -79,7 +79,7 @@ string findDMDConfig(string dmdFilePath)
     import dmd.dinifile : findConfFile;
     import std.string : fromStringz, toStringz;
 
-    auto f = findConfFile(dmdFilePath.toStringz, "dmd.conf");
+    auto f = findConfFile(dmdFilePath.toStringz[0 .. dmdFilePath.length], "dmd.conf");
```

Comment:
This is converted to a C string then back to a D string. Am I missing something?

Reply:
It was a leftover. `findConfFile` was still dependent on `\0` termination, as one of the function was not converted to `const(char)[]`. I converted it and now `\0` termination isn't required anymore.